runtime/vm/jvminit.c

```diff
@@ -3761,6 +3761,32 @@ processVMArgsFromFirstToLast(J9JavaVM * vm)
 	}
 #endif /* defined(OMR_GC_COMPRESSED_POINTERS) && defined(OMR_GC_FULL_POINTERS) */
 
+#if JAVA_SPEC_VERSION >= 18
+	{
+		/* JEP 421: Deprecate Finalization for Removal - Expecting values: "enabled" and "disabled".
+		 * The default value for now will be "enabled", * i.e., finalization is enabled by default.
```

Comment:
Please remove the extraneous `*` before 'i.e.'.

Reply:
Removed.